app/src/main/java/layout/profileFragment.java

```diff
@@ -32,6 +35,8 @@
 
     private ArrayAdapter<Mood> adapter;
 
+    private static final String frag = "profile";
+
```

Comment:
For the purposes of updating the "frag" field in FeelTripApplication, which then allows us to know which frag we're currently viewing from any point in the app.